docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -380,24 +381,42 @@ comment: |
   Get the list of fields, and check for any possible labeling errors
   right away
 code: |
+  from pdfminer.pdfparser import PDFSyntaxError
+  from PyPDF2.utils import PdfReadError
+  from zipfile import BadZipFile
+  
   try:
     if template_upload[0].extension == 'pdf':
       all_fields = get_pdf_fields(template_upload)
     else:
       all_fields = get_fields(template_upload)
       
-    if all_fields is None:
-      force_ask('empty_pdf')
+    if not all_fields:
+      empty_pdf
```

Comment:
I think the `force_ask` is necessary. Without it, when I get kicked to the event screens and press the "Restart" button, it just goes to the next screen with a toast at the top that says "Input not processed because the questions changed", and the logs say 
```
2021-06-01 22:21:17 index: not the same question name: Question_4 versus Question_8
```
Can't figure it out, but it just works with `force_ask`, so I'd stick to that.

Reply:
Weird! I'll put them back in